script/bash/verify.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
```

Comment:
Please explain why this change is made, if approved this will go into a different pull request

Reply:
Predictions woudn't work for older deployments therefore I had refactored the verify scripts to not use predicted addresses but the deployed addresses.